<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <widget class="QMenu" name="menuAR">
     <property name="title">
      <string>AR</string>
     </property>
     <addaction name="actionAR_All"/>
     <addaction name="actionAR_Attack"/>
     <addaction name="actionAR_Hit"/>
     <addaction name="actionAR_Speed"/>
     <addaction name="actionAR_Avoid"/>
     <addaction name="actionAR_Life"/>
    </widget>
    <widget class="QMenu" name="menuRF">
     <property name="title">
      <string>RF</string>
     </property>
     <addaction name="actionRF_All"/>
     <addaction name="actionRF_Attack"/>
     <addaction name="actionRF_Hit"/>
     <addaction name="actionRF_Speed"/>
     <addaction name="actionRF_Avoid"/>
     <addaction name="actionRF_Life"/>
    </widget>
    <widget class="QMenu" name="menuHG">
     <property name="title">
      <string>HG</string>
     </property>
     <addaction name="actionHG_All"/>
     <addaction name="actionHG_Attack"/>
     <addaction name="actionHG_Hit"/>
     <addaction name="actionHG_Speed"/>
     <addaction name="actionHG_Avoid"/>
     <addaction name="actionHG_Life"/>
    </widget>
    <widget class="QMenu" name="menuSG">
     <property name="title">
      <string>SG</string>
     </property>
     <addaction name="actionSG_All"/>
     <addaction name="actionSG_Attack"/>
     <addaction name="actionSG_Hit"/>
     <addaction name="actionSG_Speed"/>
     <addaction name="actionSG_Avoid"/>
     <addaction name="actionSG_Life"/>
     <addaction name="actionSG_Armor"/>
    </widget>
    <widget class="QMenu" name="menuMG">
     <property name="title">
      <string>MG</string>
     </property>
     <addaction name="actionMG_All"/>
     <addaction name="actionMG_Attack"/>
     <addaction name="actionMG_Hit"/>
     <addaction name="actionMG_Speed"/>
     <addaction name="actionMG_Avoid"/>
     <addaction name="actionMG_Life"/>
    </widget>
    <widget class="QMenu" name="menuSMG">
     <property name="title">
      <string>SMG</string>
     </property>
     <addaction name="actionSMG_All"/>
     <addaction name="actionSMG_Attack"/>
     <addaction name="actionSMG_Hit"/>
     <addaction name="actionSMG_Speed"/>
     <addaction name="actionSMG_Avoid"/>
     <addaction name="actionSMG_Life"/>
    </widget>
    <addaction name="menuAR"/>
    <addaction name="menuRF"/>
    <addaction name="menuHG"/>
    <addaction name="menuSG"/>
    <addaction name="menuMG"/>
    <addaction name="menuSMG"/>
    <addaction name="actionStatus"/>
   </widget>
   <widget class="QMenu" name="menuAdministrator">
    <property name="title">
     <string>Administrator</string>
    </property>
    <addaction name="actionlogin"/>
    <addaction name="actionNew"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSelect"/>
    <addaction name="actionlogout"/>
   </widget>
   <widget class="QMenu" name="menuMaintain">
    <property name="title">
     <string>Maintain</string>
    </property>
    <addaction name="actionDB_command"/>
   </widget>
   <widget class="QMenu" name="menuOther">
    <property name="title">
     <string>Other</string>
    </property>
    <addaction name="actionGoblin"/>
    <addaction name="actionEquip"/>
    <addaction name="actionArray"/>
    <addaction name="actionSkill"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSelect"/>
   <addaction name="menuOther"/>
   <addaction name="menuAdministrator"/>
   <addaction name="menuMaintain"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionAR_Attack">
   <property name="text">
    <string>AR_Attack</string>
   </property>
  </action>
  <action name="actionAR_Hit">
   <property name="text">
    <string>AR_Hit</string>
   </property>
  </action>
  <action name="actionAR_Speed">
   <property name="text">
    <string>AR_Speed</string>
   </property>
  </action>
  <action name="actionAR_Avoid">
   <property name="text">
    <string>AR_Avoid</string>
   </property>
  </action>
  <action name="actionAR_Life">
   <property name="text">
    <string>AR_Life</string>
   </property>
  </action>
  <action name="actionRF_Attack">
   <property name="text">
    <string>RF_Attack</string>
   </property>
  </action>
  <action name="actionRF_Hit">
   <property name="text">
    <string>RF_Hit</string>
   </property>
  </action>
  <action name="actionRF_Avoid">
   <property name="text">
    <string>RF_Avoid</string>
   </property>
  </action>
  <action name="actionRF_Speed">
   <property name="text">
    <string>RF_Speed</string>
   </property>
  </action>
  <action name="actionRF_Life">
   <property name="text">
    <string>RF_Life</string>
   </property>
  </action>
  <action name="actionHG_Attack">
   <property name="text">
    <string>HG_Attack</string>
   </property>
  </action>
  <action name="actionHG_Hit">
   <property name="text">
    <string>HG_Hit</string>
   </property>
  </action>
  <action name="actionHG_Speed">
   <property name="text">
    <string>HG_Speed</string>
   </property>
  </action>
  <action name="actionHG_Avoid">
   <property name="text">
    <string>HG_Avoid</string>
   </property>
  </action>
  <action name="actionHG_Life">
   <property name="text">
    <string>HG_Life</string>
   </property>
  </action>
  <action name="actionSG_Attack">
   <property name="text">
    <string>SG_Attack</string>
   </property>
  </action>
  <action name="actionSG_Hit">
   <property name="text">
    <string>SG_Hit</string>
   </property>
  </action>
  <action name="actionSG_Speed">
   <property name="text">
    <string>SG_Speed</string>
   </property>
  </action>
  <action name="actionSG_Avoid">
   <property name="text">
    <string>SG_Avoid</string>
   </property>
  </action>
  <action name="actionSG_Life">
   <property name="text">
    <string>SG_Life</string>
   </property>
  </action>
  <action name="actionMG_Attack">
   <property name="text">
    <string>MG_Attack</string>
   </property>
  </action>
  <action name="actionMG_Hit">
   <property name="text">
    <string>MG_Hit</string>
   </property>
  </action>
  <action name="actionMG_Speed">
   <property name="text">
    <string>MG_Speed</string>
   </property>
  </action>
  <action name="actionMG_Avoid">
   <property name="text">
    <string>MG_Avoid</string>
   </property>
  </action>
  <action name="actionMG_Life">
   <property name="text">
    <string>MG_Life</string>
   </property>
  </action>
  <action name="actionSMG_Attack">
   <property name="text">
    <string>SMG_Attack</string>
   </property>
  </action>
  <action name="actionSMG_Hit">
   <property name="text">
    <string>SMG_Hit</string>
   </property>
  </action>
  <action name="actionSMG_Speed">
   <property name="text">
    <string>SMG_Speed</string>
   </property>
  </action>
  <action name="actionSMG_Avoid">
   <property name="text">
    <string>SMG_Avoid</string>
   </property>
  </action>
  <action name="actionSMG_Life">
   <property name="text">
    <string>SMG_Life</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionDB_command">
   <property name="text">
    <string>DB_command</string>
   </property>
  </action>
  <action name="actionSG_Armor">
   <property name="text">
    <string>SG_Armor</string>
   </property>
  </action>
  <action name="actionAR_All">
   <property name="text">
    <string>AR_All</string>
   </property>
  </action>
  <action name="actionRF_All">
   <property name="text">
    <string>RF_All</string>
   </property>
  </action>
  <action name="actionHG_All">
   <property name="text">
    <string>HG_All</string>
   </property>
  </action>
  <action name="actionSG_All">
   <property name="text">
    <string>SG_All</string>
   </property>
  </action>
  <action name="actionMG_All">
   <property name="text">
    <string>MG_All</string>
   </property>
  </action>
  <action name="actionSMG_All">
   <property name="text">
    <string>SMG_All</string>
   </property>
  </action>
  <action name="actionlogin">
   <property name="text">
    <string>login</string>
   </property>
  </action>
  <action name="actionlogout">
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionGoblin">
   <property name="text">
    <string>Goblin</string>
   </property>
  </action>
  <action name="actionEquip">
   <property name="text">
    <string>Equip</string>
   </property>
  </action>
  <action name="actionArray">
   <property name="text">
    <string>Array</string>
   </property>
  </action>
  <action name="actionSkill">
   <property name="text">
    <string>Skill</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
